lib/peep.rb

```diff
@@ -6,7 +6,7 @@ class Peep
   def self.all
     connection = PG.connect(dbname: 'chitter')
     result = connection.exec("SELECT * FROM peeps;")
-    result.map { |peep| peep['message']}
+    result.map { |peep| peep['message'] }.reverse
```

Comment:
I used a similar method, apparently another way to look at it would be like in Line 8
SELECT * FROM peeps DESC; (syntax might be a little off but this reverses the list order too)